docs: add crd page for consistency (#824)
redpanda-data/documentation: docs/platform/reference/redpanda-operator/crd.mdx

---
title: Operator Custom Resource Definition (CRD)
---

<head>
    <meta name="title" content="Operator Custom Resource Definition (CRD)| Redpanda Docs"/>
    <meta name="description" content="Operator Custom Resource Definition (CRD)."/>
</head>

> Redpanda offers a Operator Custom Resource Definition (CRD). 

Reviewer: ```suggestion
Redpanda offers a CRD for the Redpanda Operator. 
```

> You can check the file at [Operator Custom Resource Definition reference page](https://doc.crds.dev/github.com/vectorizedio/redpanda).

Reviewer: ```suggestion
See the [Redpanda Operator Custom Resource Definition](https://doc.crds.dev/github.com/vectorizedio/redpanda).
```
